pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -55,37 +61,63 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="mesos"></a>Mesos Permissions
 
+It is important to note that the principal issuing the command to Mesos is the
+one that must be authorized to do so. This is not always the DC/OS user that is
+logged into the UI or CLI. For example, when a DC/OS user uses the CLI or UI to
+launch a Marathon application the Marathon framework performs its own
+authorization of that user before deciding to create the application. Once
+Marathon has determined that the DC/OS user is permitted to create the
+application, Marathon uses its own DC/OS service account user called
+`dcos_marathon` to instruct Mesos to launch the actual Mesos tasks. At that
+point, Mesos will check that the `dcos_marathon` service account (and not the
+end user) is authorized to perform a `create` action on the
+`dcos:mesos:master:task:app_id` resource.
+
+Permission string sections for `<role-name>`s in square brackets are optional.
```

Comment:
This section about the roles related to `slave_public` was copied from the 1.11 section. It looked true to me, @jgehrcke @vespian can you confirm?

Reply:
lgtm

@sambatyon ?